http-kernel-fixtures/sub-folder/cookie_page4.php

```diff
@@ -0,0 +1,19 @@
+<?php
+    $resp = new Symfony\Component\HttpFoundation\Response();
+    if (method_exists('Symfony\Component\HttpFoundation\Cookie', 'create')) {
+        $cookie = Symfony\Component\HttpFoundation\Cookie::create('srvr_cookie', 'srv_var_is_set', 0, '/sub-folder/');
+    } else {
+        $cookie = new Symfony\Component\HttpFoundation\Cookie('srvr_cookie', 'srv_var_is_set', 0, '/sub-folder/');
```

Comment:
Using path from DocumentRoot would only work when running a test suite with PHP built-in web server.

Somehow the `$_SERVER['WEB_FIXTURES_HOST']` needs to be incorporated (when present) to support running test suite in existing Web Server's sub-folder.